dashboard/app/views/sections/new.html.haml

```diff
@@ -1,4 +1,4 @@
-%script{src: webpack_asset_path('js/sections/new.js'), data: {isUsersFirstSection: @is_users_first_section.to_json, canEnableAITutor: current_user.can_enable_ai_tutor?.to_json}}
+%script{src: webpack_asset_path('js/sections/new.js'), data: {isUsersFirstSection: @is_users_first_section.to_json, canEnableAITutor: current_user.can_enable_ai_tutor?.to_json, showChildAccountPolicy: %w[US RD].include?(@user_country).to_json}}
```

Comment:
I feel like the user country logic could be abstracted somewhere else since we check the user's country in multiple places for Child Account Policy purposes.